hypha/http.py

```diff
@@ -792,7 +792,14 @@ async def liveness(req: Request) -> JSONResponse:
             """Used for liveness probe.
             If the liveness probe fails, it means the app is in a failed state and restarts it.
             """
-            return JSONResponse({"status": "OK"})
+            try:
+                await connect_to_server({
```

Comment:
Instead of doing connect to server, maybe we can check the redis connection instead. The previous crash was mainly caused by time out error of the manager service. I will propose something later.

If we go for connect to server, it's better to use async with so we close the connection properly.